### ISSUE TITLE
Reviewing cultural form instance relations

### ISSUE THREAD

**Issue by alliyya** [cwrc/ontology#425]
Review cultural form instances for missing connections between terms
Within in that class, skos:broaderTransitive/skos:related
And out of that class. 
Adding relations to cwrc:TextualLabels, possibly creating new ones as we see fit.

**Comment by lemaka** [comment]
Is this something I can help with, @alliyya ?

**Comment by JasmineDW** [comment]
@SusanBrown @alliyya  This appears to be implemented within the rdf file. Was this completed? or tracked anywhere so we can pick this up again?

**Comment by alliyya** [comment]
I don't think this was tracked anywhere usefully, I think some initial relationships were put in as terms were added. This was just a "would be nice" if the terms were examined for possible relationships to other existing terms so that we can add it to enhance future info retrieval. 

I think we have decent broaderTransitive/narrowTransitive relations for Religion but not other cultural forms. We may want to add some skos:related relationships between occupations and social class, ex. cwrc:shopkeepers and cwrc:retail

cwrc:shopkeepers
--


<br class="Apple-interchange-newline">

cwrc:shopkeepers
--


<br class="Apple-interchange-newline">
